src/webportal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pai-webportal",
-  "version": "0.0.0",
+  "version": "0.14.0",
```

Comment:
the version is labeled [here](https://github.com/microsoft/pai/blob/master/src/webportal/build/build-pre.sh#L25) before building webportal